lcls-twincat-sample-delivery/Library/POUs/VICI/FB_ViciDriver.TcPOU

```diff
@@ -100,20 +100,10 @@ CASE iStep OF
         (* Mirror the "clock" value across the vertical axis of symmetry *)
         iReqPosReverse := 13 - i_stControl.iReqPos;
         IF q_stStatus.iReqPos <> i_stControl.iReqPos THEN
-            IF q_stStatus.iCurrPos > i_stControl.iReqPos THEN
-                IF ABS(q_stStatus.iCurrPos - i_stControl.iReqPos) >= 6 THEN
-                    sSendData := CONCAT('CW', INT_TO_STRING(iReqPosReverse));
-                ELSE
-                    sSendData := CONCAT('CC', INT_TO_STRING(iReqPosReverse));
-                END_IF
-            END_IF
-            IF q_stStatus.iCurrPos < i_stControl.iReqPos THEN
-                IF ABS(q_stStatus.iCurrPos - i_stControl.iReqPos) >= 6 THEN
-                    sSendData := CONCAT('CC', INT_TO_STRING(iReqPosReverse));
-                ELSE
-                    sSendData := CONCAT('CW', INT_TO_STRING(iReqPosReverse));
-                END_IF
-            END_IF
+
+            // Move actuator to new position via the shortest path
+            sSendData := CONCAT('GO', INT_TO_STRING(iReqPosReverse));
+
```

Comment:
The code change here is pretty straightforward and you have tested in prod, this is an easy approve if not for the weird extra files included in the commit